x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/threshold_rule.cy.ts

```diff
@@ -117,6 +126,7 @@ describe('Threshold rules', { tags: ['@ess', '@serverless'] }, () => {
         'have.text',
         `Results aggregated by ${rule.threshold.field} >= ${rule.threshold.value}`
       );
+      assertDetailsNotExist(SUPPRESS_FOR_DETAILS);
```

Comment:
I'm not a fan of having assertions wrapped in methods. Usually for debugging purposes or when a test fails, is better to have the assertion directly on the test :)

Reply:
This utility is used widely across code base already, that's why I used it.
It encapsulates some logic
```
export const assertDetailsNotExist = (title: string | RegExp) =>
  cy.get(DETAILS_TITLE).contains(title).should('not.exist');
```

that makes less code boilerplate used.

Do we want to replace it with just pure assertion? Maybe separate PR would be better place to do it, since it has more than dozen usages across our code base?

Reply:
I understand and yes, that should not be part of this PR specially when is happening across tests and you are just reusing a method :)